DOCS/man/mpv.rst

```diff
@@ -39,9 +39,11 @@ LIRC support - configure remotes as input devices instead).
 
 See the ``--input-`` options for ways to customize it.
 
-The following listings are not necessarily complete. See ``etc/input.conf`` for
-a list of default bindings. User ``input.conf`` files and Lua scripts can
-define additional key bindings.
+The following listings are not necessarily complete. See ``etc/input.conf`` 
+in the github repo for a list of default bindings. User ``input.conf`` files 
+and Lua scripts can define additional key bindings.
+
+See COMMAND INTERFACE and KEY NAMES sections for more details on configuring keybindings.
```

Comment:
"COMMAND INTERFACE" and "Key names" (note the capitalization) are references to existing sections elsewhere, and so they should be encosed in single back-ticks followed by underscore (see elsewhere at this file), so that various renditions of the docs can cross-link them correctly.